src/docs/contributing/linking.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Linking Variables
+noindex: true
+sidebar_order: 4
+---
+
+We create a variety of links in our documentation, but the most commonly-used link type connects content either between product guides and the platform pages or between platform pages themselves. Each of these may take a different format, depending on whether a platform needs to be specified.
+
+### From Product to Platform
+
+We use this format when linking from the product guides to the platform content because it allows the user to specify their platform if it is not already specified. For example:
```

Comment:
```suggestion
We use this format when linking from the product guides to the platform content because it allows the user to specify their platform, if it is not already specified. For example:
```